converters/fromZigbee.js

```diff
@@ -5686,6 +5691,55 @@ const converters = {
             return payload;
         },
     },
+    silvercrest_smart_led_string: {
+        cluster: 'manuSpecificTuya',
+        type: ['commandGetData', 'commandSetDataResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const dp = msg.data.dp;
+            const value = tuyaGetDataValue(msg.data.datatype, msg.data.data);
+
+            if (dp === common.TuyaDataPoints.silvercrestChangeMode) {
+                meta.state.mode = value;
```

Comment:
If I understand correctly `value` is an int here, this is not self-explaining and string are preferred here. However I'm wondering if we should completely drop this mode, we don't do this for other lights.

Reply:
These lights have 3 modes which are in common.silvercrestModes. The device actually needs this mode set before it will act acording to that mode. (So white mode is needed  for white light / brightness, color mode is needed for being able to change HSB, scene mode is needed for the scenes)

It could maybe be used to skip setting the mode when you see in the meta.state that it's already there. The Lidl gateway does that.
But it doesn't hurt to just set the mode before giving it a command anyway, so I agree we could just remove it from emta.state.

Reply:
Decided to remove it.